examples/rust/tcp-dump.rs

```diff
@@ -45,27 +45,16 @@ pub const AF_INET: i32 = libc::AF_INET;
 #[cfg(target_os = "linux")]
 pub const SOCK_STREAM: i32 = libc::SOCK_STREAM;
 
-//======================================================================================================================
-// Program Arguments
-//======================================================================================================================
-
-/// Program Arguments
 #[derive(Debug)]
 pub struct ProgramArguments {
-    /// Local socket IPv4 address.
-    local: SocketAddr,
+    local_addr: SocketAddr,
```

Comment:
do you want to label this local_ip_addr? Just to be clear because there are lots of different types of addresses and it's the one that we use in the inet stack.

Reply:
Yes, that sounds good. Will updated.